README.md

```diff
@@ -210,12 +209,12 @@ $ rails generate react:component App
 
 ### Making authenticated requests using App-Bridge
 
-This section assumes that your app meets the following prerequisities:
+This section assumes that your app meets the following prerequisites: 
 
 * Your app has the beta feature `app_session_token` enabled
```

Comment:
do we want to make this public? It resides on core and gives away the specific beta flag name that we use internally. Could we possibly say, `Your app has the next-gen auth beta feature enabled on Shopify`?

Reply:
Good call! Updating in next push